openstack_controller/datadog_checks/openstack_controller/metrics.py

```diff
@@ -354,6 +354,8 @@
 CINDER_METRICS_PREFIX = "openstack.cinder"
 CINDER_SERVICE_CHECK = f"{CINDER_METRICS_PREFIX}.api.up"
 CINDER_RESPONSE_TIME = f"{CINDER_METRICS_PREFIX}.response_time"
+CINDER_VOLUME_COUNT = f"{CINDER_METRICS_PREFIX}.volume.count"
```

Comment:
Can you update the metadata.csv with this new metric?

Reply:
Added.